components/brave_rewards/resources/page/components/icons/caret_icon.tsx

```diff
@@ -0,0 +1,30 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+
+interface Props {
+  direction: 'up' | 'down' | 'left' | 'right'
+}
+
+function rotation (direction: string) {
+  switch (direction) {
+    case 'left': return 180
+    case 'down': return 90
+    case 'up': return -90
+    default: return 0
+  }
+}
+
+export function CaretIcon (props: Props) {
```

Comment:
Since this is a one-off caret (instead of a general-purpose one like in "shared"), I think maybe we should name this `SelectProviderCaret` and drop the `direction` prop (since it will always just point the same way).

Reply:
We had a conversation about this caret icon in Slack. We'd like to move over to this one in the future, hence the `direction` prop. Do you want me to rename it and make these changes in the meantime?

Reply:
Yes, I think so.